.circleci/config.yml

```diff
@@ -0,0 +1,54 @@
+version: 2.1
+
+executors:
+  my-docker-executor:
+    docker:
+      - image: cimg/python:3.11.10
+
+jobs:
+  build:
+    executor: my-docker-executor
+    steps:
+      - checkout
+
+      - run:
+          name: Get uv via curl and install it
```

Comment:
The project is not containerized as of now. And for the CI part, we just need to install dependencies to verify our code standards (with formatting and linting) and functionality (with atleast unit testing).

I chose the image provided by circleCI as it would be most optimized for their systems.